spec/features/admins/invoices/show_spec.rb

```diff
@@ -29,7 +29,7 @@
 
     expect(page).to have_content(@invoice_1.id)
     expect(page).to have_field(:status)
-    expect(page).to have_content('Saturday Nov 6, 2021')
+    expect(page).to have_content(@invoice_1.created_at.strftime('%A, %B %d, %Y') )
```

Comment:
Thank you! I now realize that the invoice gets "created" when we run the test, so this will totally work.